toolchain/check/testdata/function/builtin/types.carbon

```diff
@@ -0,0 +1,203 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+// --- types.carbon
+
+library "types" api;
+
+fn Int() -> type = "int.make_32";
+fn Float(size: i32) -> type = "float.make";
+fn Bool() -> type = "bool.make";
```

Comment:
I wonder if `"float.type"` or `"float.maketype"` would be a better builtin name than `"float.make"` -- the latter sounds like a function `Float.Make` which I'd expect to produce a floating-point value, not a floating-point type.

But I'm happy with anything here; these names are just an implementation detail after all.